tests/phpunit/PaymentMethods/WC_Stripe_UPE_Payment_Gateway_Test.php

```
@@ -3847,4 +3847,42 @@ public function provide_test_is_valid_optimized_checkout_page() {
+				'expected' => '<strong>Test mode:</strong> use card <button type="button" class="js-wc-stripe-copy-test-number" aria-label="' . $copy_label . '" title="' . $copy_label . '"><i></i><span>4242 4242 4242 4242</span></button> with any expiry.',
+			],
+			'string without number tag' => [
+				'input'    => '<strong>Test mode:</strong> use any 6-digit number.',
+				'expected' => '<strong>Test mode:</strong> use any 6-digit number.',
+			],
+			'empty string' => [
+				'input'    => '',
+				'expected' => '',
+			],
+		];
+	}
```

PHPMD (2.15.0)

[error] The method provider_expand_copy_button_markup is not named in camelCase. (undefined)

(CamelCaseMethodName)